web-app/js/portal/data/MetadataRecordFetcher.js

```diff
@@ -40,7 +38,7 @@ Portal.data.MetadataRecordFetcher = Ext.extend(Ext.util.Observable, {
 
             if (metadataRecord) {
                 _this.dataCollectionStore.add(dataCollection);
-                Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_DATA_COLLECTION, dataCollection);
+                viewport.setActiveTab(TAB_INDEX_VISUALISE);
```

Comment:
I agree. However, it is something we do already in code [citation needed] so it isn't introducing any new bad practices. It'd be great if we think of something to replace both of these instances!